prediction_market_agent_tooling/markets/omen/data_models.py

```
@@ -1,8 +1,8 @@
 import typing as t
 from datetime import datetime
-
+from typing_extensions import Self
```

Ruff

`typing_extensions.Self` imported but unused

Remove unused import: `typing_extensions.Self`

(F401)